Remotion/Web/Core/UI/SmartPageEvents.cs

```diff
@@ -34,8 +34,6 @@ public enum SmartPageEvents
     OnAbort,
     /// <summary> Raised when the user scrolls the page. Signature: <c>void Function ()</c> </summary>
     OnScroll,
```

Comment:
I think we still need to create the issue to remove on scroll?

Reply:
https://re-motion.atlassian.net/browse/RM-8717